app.json

```diff
@@ -1,4 +1,5 @@
 {
   "name": "RocketChatRN",
-  "displayName": "RocketChatRN"
+  "displayName": "RocketChatRN",
+  "server": "https://ilarion.rocket.chat"
```

Comment:
Change to open.

Reply:
test will start failing failling I cannot do anything

Reply:
You mean e2e? Don't worry.

Reply:
done